Dockerfile

```diff
@@ -31,7 +33,17 @@ RUN echo "@community http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /et
     && rm -rf /tmp/* /var/tmp/* \
     && groupmod -g 1000 users \
     && useradd -u 911 -U -d /config -s /bin/false abc \
-    && usermod -G users abc
+    && usermod -G users abc \
+    && addgroup -S privoxy && adduser -S privoxy -G privoxy \
+    && mkdir /opt/privoxy \
+    && curl -sL https://www.privoxy.org/sf-download-mirror/Sources/3.0.29%20%28stable%29/privoxy-3.0.29-stable-src.tar.gz | tar -C /opt/privoxy --strip-components=2 -xz \
+    # && git clone https://www.privoxy.org/git/privoxy.git /opt/privoxy \
```

Comment:
comment should be removed

Reply:
I expect this comment is a blocker because the shell doesn't interpret the `\` at the end of the line as a continuation.

Reply:
Worked fine for me but I appreciate may not for others so removed :)